templates/docs/examples/patterns/rich-list-horizontal/ordered-responsive.html

```diff
@@ -0,0 +1,77 @@
+        <img class="p-image-container__image" src="https://assets.ubuntu.com/v1/a299c914-GettyImages-DataCenter.jpeg" alt="">
+      </div>
+    {%- endif -%}
+    {%- if slot == 'logo_section' -%}
+      <div class="p-logo-section">
+        <div class="p-logo-section__items">
+          <div class="p-logo-section__item">
+            <img src="https://assets.ubuntu.com/v1/e44aa9b2-nvidia-logo.png" alt="NVIDIA logo" class="p-logo-section__logo" />
+          </div>
+          <div class="p-logo-section__item">
+            <img src="https://assets.ubuntu.com/v1/94b00f2b-intel-new-logo.png" alt="Intel logo" class="p-logo-section__logo" />
+          </div>
+          <div class="p-logo-section__item">
+            <img src="https://assets.ubuntu.com/v1/e4bbd168-amd-logo.png" alt="AMD logo" class="p-logo-section__logo" />
+          </div>
+          <div class="p-logo-section__item">
+            <img src="https://assets.ubuntu.com/v1/b0404df4-hp-logo.png" alt="HP logo" class="p-logo-section__logo" />
+          </div>
+          <div class="p-logo-section__item">
+            <img src="https://assets.ubuntu.com/v1/90161e97-dell-logo.png" alt="Dell logo" class="p-logo-section__logo" />
+          </div>
+          <div class="p-logo-section__item">
+            <img src="https://assets.ubuntu.com/v1/35549e17-lenovo-logo.png" alt="Lenovo logo" class="p-logo-section__logo" />
+          </div>
+          <div class="p-logo-section__item">
+            <img src="https://assets.ubuntu.com/v1/94b00f2b-intel-new-logo.png" alt="Intel logo" class="p-logo-section__logo" />
+          </div>
+        </div>
+      </div>
+    {%- endif -%}
+    {%- if slot == 'description' -%}
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
+        aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate.
+      </p>
+    {%- endif -%}
+    {%- if slot == 'list_item_1' -%}
+      <a href="#">Active Directory</a> and LDAP integration
+    {%- endif -%}
+    {%- if slot == 'list_item_2' -%}
+      Full disk encryption
+    {%- endif -%}
+    {%- if slot == 'list_item_3' -%}
+      Security updates for the <a href="#">full open source stack</a>
+    {%- endif -%}
+    {%- if slot == 'list_item_4' -%}
+      Estate <a href="#">monitoring and management</a>
+    {%- endif -%}
+    {%- if slot == 'list_item_5' -%}
+      Configuration lockdown
+    {%- endif -%}
+    {%- if slot == 'list_item_6' -%}
+      Local Kubernetes for offline development
+    {%- endif -%}
+    {%- if slot == 'list_item_7' -%}
+      Instant Ubuntu on Windows and macOS with <a href="#">WSL</a> and <a href="#">Multipass</a>
+    {%- endif -%}
+    {%- if slot == 'cta' -%}
+      <p>
+        <a href="#" class="p-button">Learn more</a>
+        <a href="#">Contact us &rsaquo;</a>
```

Comment:
```suggestion
        <a href="#">Contact us&nbsp;&rsaquo;</a>
```
We normally use the non-breaking space entity for CTA actions. Can you update the examples to reflect this change too please?

Reply:
Good call, fixed!